.travis.yml

```diff
@@ -25,8 +24,6 @@ matrix:
       env: "CHECK=rubocop"
     - rvm: 2.0.0
       env: "CHECK=rubocop"
-    - rvm: 1.9.3
-      env: "CHECK=rubocop"
```

Comment:
We test against 1.9.3 to ensure that that puppet doesn't introduce ruby language features that aren't compatible with JRuby 1.7 (used in puppetserver 2.8.x). Can we not drop this check?

Reply:
@joshcooper packaging doesn't work with ruby < 2.0, so we'll have to figure something out

Reply:
Ideally bundler would ignore the `packaging` gem in the `Gemfile` when using `--without ... packaging`, but I seem to recall a bundler bug where it still includes it during dependency resolution. Is that what you're seeing @mwaggett?

Reply:
ahh, I didn't know travis could exclude, will add that and see what happens

Reply:
so yeah @joshcooper I think the current travis error is what you're talking about